src/templatizer.js

```diff
@@ -17,25 +17,29 @@ const templatizer = (context, config) => {
         switch (argumentName) {
             case '--version':
                 optionSchema.id = customPrefix + 'version';
-                optionSchema.type = 'null';
+                option.type = 'null';
                 break;
             case '--help':
                 optionSchema.id = customPrefix + 'help';
-                optionSchema.type = 'null';
+                option.type = 'null';
                 break;
             case '--config':
                 optionSchema.id = customPrefix + 'config';
+                option.type = 'string';
                 break;
             case '--stdin':
                 optionSchema.id = customPrefix + 'stdin';
+                option.type = 'string';
```

Comment:
Why do you use `option.type`, but not `optionSchema.type` ?

Reply:
@svoboda-rabstvo because it's assigning later for **all** arguments. But, yes, maybe it is better to do it at once. I will change it.